SPECS/mstflint/mstflint.spec

```
@@ -29,9 +33,17 @@ Requires:       python3
 This package contains firmware update tool, vpd dump and register dump tools
 for network adapters based on Mellanox Technologies chips.
 
+%package kernel
+Summary:        mstflint Kernel Modules
+Group:          System Environment/Kernel
+Requires:       kernel >= 5.15.0 and kernel < 5.16.0
+Requires:       kernel-headers >= 5.15.0 and kernel-headers < 5.16.0
```

Similar issue with our tooling not supporting `and`:
```suggestion
Requires:       kernel >= 5.15.0
Requires:       kernel < 5.16.0
Requires:       kernel-headers >= 5.15.0
Requires:       kernel-headers < 5.16.0
```

---

@PawelWMS  If it doesn't support why didn't throw any error? I expect RPM version > 4.13 support it.

---

It's not about RPM, it's about our own build tooling that parses the specs and builds a dependency diagram. RPM definitely supports that. It's possible that it supports some of these keywords, but to be on the safe side, I wanted to go with this option.

---

Yeah, the tooling is too dumb and fails with issues like these:
```
(...)
INFO[0271] ---------------------------                  
INFO[0271] --------- Summary ---------                  
INFO[0271] ---------------------------                  
INFO[0271] Number of built SRPMs:             1         
INFO[0271] Number of prebuilt SRPMs:          0         
INFO[0271] Number of failed SRPMs:            0         
INFO[0271] Number of blocked SRPMs:           0         
INFO[0271] Number of unresolved dependencies: 1 
INFO[0271] Built SRPMs:                                 
INFO[0271] --> mstflint-4.21.0-4.cm2.src.rpm            
INFO[0271] Unresolved dependencies:                     
INFO[0271] --> and:C:''V:'',C2:''V2:'
...

For now we'd need to stick to separate lines.